src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -466,14 +493,23 @@ export default class ExtraproRegistration extends Mixins(DateMixin, DocumentMixi
     return (this.business.status === EntityStates.ACTIVE)
   }
 
-  readonly businessNameRules: Array<VuetifyRuleFunction> = [
+  readonly homeIdentifierRules: Array<VuetifyRuleFunction> = [
+    (v) => !!v || 'Registration is required',
+    (v) => (v && v.length <= 50) || 'Cannot exceed 50 characters'
+  ]
+
+  readonly homeLegalNameRules: Array<VuetifyRuleFunction> = [
     (v) => !!v || 'Name is required',
     (v) => (v && v.length <= 1000) || 'Cannot exceed 1000 characters'
   ]
 
-  readonly incorporationDateRules: Array<VuetifyRuleFunction> = [
-    (v) => !!v || 'Incorporation Date is required'
-  ]
+  get incorporationDateRules (): Array<VuetifyRuleFunction> {
+    return [
+      (v) => !!v || 'Incorporation Date is required',
+      () => (this.business.homeIncorporationDate <= this.getCurrentDate) ||
+        'Incorporation Date cannot be in the future'
+    ]
```

Comment:
A couple of notes here:
- This is now a getter (instead of a static/readonly array) because it uses variables in the second calculation.
- Although the date picker's max date is today, this is a safety check in case the data was somehow manipulated. I implemented the same thing in the other components.